data/manifest_core.json

```diff
@@ -746,6 +746,13 @@
           "default": "NA"
         }
       },
+      "yellow-fever": {
```

Comment:
Suggestion: remove `yellow-fever` from this comment

https://github.com/nextstrain/nextstrain.org/blob/a5d9b064003ab53a0c540cc1bbe13b22823c0eab/src/routing/core.js#L30-L31

Reply:
Removed; updated count and date.